src/app/page.js

```diff
@@ -1,8 +1,15 @@
+import NavBar from '../components/NavBar';
+import Footer from '../components/footer/Footer';
+
 export default function Home() {
   return (
-    <main className='flex flex-col items-center justify-between p-24 gap-4'>
-      <h1 className='text-5xl font-bold'>Braguitas Handmade</h1>
-      <p className='text-2xl font-medium'>Braguitas hechas con amor</p>
+    <main>
+      <NavBar />
+      <div className='flex flex-col items-center justify-between p-24 gap-4'>
+        <h1 className='text-5xl font-bold'>Braguitas Handmade</h1>
+        <p className='text-2xl font-medium'>Braguitas hechas con amor</p>
+      </div>
+      <Footer />
     </main>
```

Comment:
It makes more sense semantically to leave the NavBar and the Footer outside of `<main>`, since these are a `<header>` and a `<footer>` respectively. And leaving them in `layout.js` should be the correct thing to do, just because it's a reusable component that is visible throughout the page, and layout should render in every single one.